tests/testthat/test-convert_year_types.R

```diff
@@ -0,0 +1,61 @@
+test_that("Can convert a year from normal financial year to the alternate form", {
```

Comment:
Tests are good.
No need to test all years, one or two is fine. Testing edge cases is good though i.e. earliest file = 2011/12.  Also, test incorrect inputs e.g. what if we run `convert_fyyear_to_year("2017)`?

Reply:
removed unnecessary year tests

Would I include an error message to make sure that the correct form has been inputted?